install/kubernetes/cilium/templates/NOTES.txt

```diff
@@ -1,4 +1,10 @@
-You have successfully installed {{ title .Chart.Name }}.
+{{- if (and (.Values.preflight.enabled) (not (.Values.agent.enabled)) (not (.Values.config.enabled)) (not (.Values.operator.enabled))) }}
+    You have successfully completed the prefight check.
+{{- else if (not (.Values.config.enabled))}}
+    You have preserved the configMap and successfully installed {{ title .Chart.Name}}.
+{{- else }}
+    You have successfully installed {{ title .Chart.Name }}.
```

Comment:
Similarly, I wonder if `installed` here would be more accurately replaced with `configured` (though it's possible that the configuration was still unsuccessful if, for instance, an unsupported set of flags are configured). Hmm. I wonder what kind of message that other open source projects puts here, do they also say "installed" even though we don't know if the install was successful? Or do they drop the `successfully` qualifier so the user knows they've set this version up, but it's up to them to confirm whether it was successful?

Reply:
I think at this point the helm's responsibility is just to install the given the charts. The validity of the installation should be done later. Since we are already pointing to the documentation, we hope that things should be fine.

Reply:
That's probably fair. I'll mark this comment as resolved and if someone still thinks it's confusing in future then we can further improve at that point.